Bar chart changes (#212)
patternfly/training-scenarios: react-charts/bar-chart/step1.md

This step will begin by adding a CSS selector to define the chart size.

> <strong>Note:</strong> Katacoda is setting up a new React application. It will be ready to code

Author: updated from your PR

Begin by adding a CSS selector to define the chart size.

Katacoda is setting up a new React application. Begin coding once the server
starts and "Welcome to PatternFly" is visible on the lower pane.

<strong>Note:</strong> Katacoda is setting up a new React application.

> It's ready to code once the server starts and "Welcome to PatternFly" message

Reviewer: ```suggestion
It's ready to code once the server starts and "Welcome to PatternFly" message
```
```suggestion
It's ready to code once the server starts and "Welcome to PatternFly" message
```
```suggestion
It is ready to code once the server starts and "Welcome to PatternFly" message
```

> It's ready to code once the server starts and "Welcome to PatternFly" message

Reviewer: ```suggestion
It is ready to code once the server starts and "Welcome to PatternFly" message
```

is displayed on the lower pane.

1) <strong>Navigate to the `src` folder and open `src/app.css`{{open}}</strong>

2) <strong>Add CSS to define the overall chart size.</strong>

Copy the following code into the `app.css` file, replacing all of the content there:

<pre class="file" data-filename="src/app.css" data-target="replace">
.chart-container {
  height: 250px;
  width: 600px;
}
</pre>

> <strong>Note: </strong>Victory renders responsive containers, so the chart's

Reviewer: ```suggestion
<strong>Note: </strong>Victory renders responsive containers, so that the chart's
```
